test/commands/distro/distro.withdraw.test.ts

```
@@ -0,0 +1,235 @@
+import { expect } from 'chai'
+import { runCli } from '../../runCli'
+
+// Helper to strip ANSI color codes
+const stripAnsi = (str: string) => str.replace(/\u001b\[\d+m/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)